### ISSUE TITLE
Jupyter: move cell up/down buttons

### ISSUE THREAD

**Issue by postrational** [microsoft/vscode-jupyter#6812]
In the Jupyter plugin, I cannot find "Move cell up" and "Move cell down" buttons.

Where I would expect to see them, I find  "execute cell and below/above" buttons, which are confusing and in my opinion unnecessary.

Drag-and-drop reordering is nice, but when you have tall cells with long output, it's much harder to use than clicking a button.

Edit: also, a "Run next cell" button would be nice in the top interface, so we can run 3 cells by clicking it 3 times, instead of pecking with the mouse for the next "Execute cell". (I am aware of keyboard shortcut)

**Comment by joyceerhl** [comment]
Thanks for the request. As a workaround for now, you might find the commands to move cells helpful:

![image](https://user-images.githubusercontent.com/30305945/127107696-af3caf2a-1757-44d0-a619-ae4e82c03037.png)

**Comment by juhap** [comment]
You can select the cell by clicking on the left, outside the text entry area.  You can also select multiple cells by using either ctrl or shift (Windows) while selecting them.

You can either use the shortcut to move selected cells or drag them.

**Comment by wwdok** [comment]
> You can select the cell by clicking on the left, outside the text entry area. You can also select multiple cells by using either ctrl or shift (Windows) while selecting them.
> 
> You can either use the shortcut to move selected cells or drag them.

Yes, i found if i put my mouse on the left of cell outside the text entry area, and click left mouse to activate current cell, then i can drag and drop this cell, but for those who are new to VScode jupyter notebook, maybe they don't know this. You say we can use shortcut to do this, but how to set the shortcut ?

**Comment by nathanneuro** [comment]
I would prefer up/down buttons instead of drag. Drag is tricky with long notebooks & large cells.

**Comment by YoraiLevi** [comment]
> Alt (option)+ DOWN/UP has often a problem with focusing on cell - it moves line up/down instead of a cell, is not reliable, I need to use a command launcher, its not good

Good to know this even exists. it wasn't intuitive at all.

**Comment by imaspol** [comment]
The shortcut works only for one cell, although the command palette workaround works for multiple chosen cells. Drag is functioning, but jumping over multiple cells is complicated: it won't scroll sometimes.
Version : 1.69.2 (system setup)
Validation : 3b889b090b5ad5793f524b5d1d39fda662b96a2a
Date : 2022-07-18T17:05:44.765Z
Electron : 18.3.5
Chromium: 100.0.4896.160
Node.js : 16.13.2
V8 : 10.0.139.17-electron.0
Système d’exploitation : Windows_NT ia32 10.0.22000

**Comment by kieferrm** [comment]
I think the way to address the issue is to provide configurability of the cell toolbar, particularly given that we try to make all toolbars more easily configurable across VS Code.

**Comment by bagcifurkan** [comment]
Alt + Up/down seems OK for me. Additionally, it would be great to add to the cell toolbar.
